### PR TITLE
[KOGITO-9029] Rocksdb addon for quarkus

### PR DESCRIPTION
**Pull request by fjtirado**
Rockds db addon for quarkus. 
See modified example for testing it https://github.com/kiegroup/kogito-examples/pull/1648

### CODE REVIEW

**Comment by kie-ci3** [comment]
**(tests) - serverless-workflow-examples job** `#763` was: **UNSTABLE**
Possible explanation: This should be test failures


<details>
<summary><b>Reproducer</b></summary>


export BUILD_MVN_OPTS_CURRENT= 
build-chain build cross_pr -f 'https://raw.githubusercontent.com/kiegroup/kogito-pipelines/main/.ci/pull-request-config.yaml' -o 'bc' -p kiegroup/kogito-examples -u https://github.com/kiegroup/kogito-runtimes/pull/2946 --skipParallelCheckout

NOTE: To install the build-chain tool, please refer to https://github.com/kiegroup/github-action-build-chain#local-execution

</details>


Please look here: https://eng-jenkins-csb-business-automation.apps.ocp-c1.prod.psi.redhat.com/job/KIE/job/kogito/job/main/job/pullrequest/job/kogito-runtimes.tests.downstream.serverless-workflow-examples/763/display/redirect

**Test results:**
- PASSED: 77
- FAILED: 1

Those are the test failures: 
<details>
<summary><a href="https://eng-jenkins-csb-business-automation.apps.ocp-c1.prod.psi.redhat.com/job/KIE/job/kogito/job/main/job/pullrequest/job/kogito-runtimes.tests.downstream.serverless-workflow-examples/763/testReport/org.kie.kogito.examples/DataIndexRestIT/testDataIndexRest/">org.kie.kogito.examples.DataIndexRestIT.testDataIndexRest</a></summary>
1 expectation failed.<br/>JSON path data.ProcessInstances[0].state doesn't match.<br/>Expected: is "COMPLETED"<br/>  Actual: ACTIVE<br/>
</details>

**Comment by kie-ci3** [comment]
**(tests) - serverless-workflow-examples job** `#765` was: **UNSTABLE**
Possible explanation: This should be test failures


<details>
<summary><b>Reproducer</b></summary>


export BUILD_MVN_OPTS_CURRENT= 
build-chain build cross_pr -f 'https://raw.githubusercontent.com/kiegroup/kogito-pipelines/main/.ci/pull-request-config.yaml' -o 'bc' -p kiegroup/kogito-examples -u https://github.com/kiegroup/kogito-runtimes/pull/2946 --skipParallelCheckout

NOTE: To install the build-chain tool, please refer to https://github.com/kiegroup/github-action-build-chain#local-execution

</details>


Please look here: https://eng-jenkins-csb-business-automation.apps.ocp-c1.prod.psi.redhat.com/job/KIE/job/kogito/job/main/job/pullrequest/job/kogito-runtimes.tests.downstream.serverless-workflow-examples/765/display/redirect

**Test results:**
- PASSED: 77
- FAILED: 1

Those are the test failures: 
<details>
<summary><a href="https://eng-jenkins-csb-business-automation.apps.ocp-c1.prod.psi.redhat.com/job/KIE/job/kogito/job/main/job/pullrequest/job/kogito-runtimes.tests.downstream.serverless-workflow-examples/765/testReport/org.kie.kogito.examples/DataIndexRestIT/testDataIndexRest/">org.kie.kogito.examples.DataIndexRestIT.testDataIndexRest</a></summary>
1 expectation failed.<br/>JSON path data.ProcessInstances[0].state doesn't match.<br/>Expected: is "COMPLETED"<br/>  Actual: ACTIVE<br/>
</details>

File: quarkus/addons/persistence/rocksdb/deployment/src/main/java/org/kie/kogito/persistence/quarkus/rocksdb/KogitoAddOnPersistenceRocksDBProcessor.java [hunk 1]
```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright 2021 Red Hat, Inc. and/or its affiliates.
```
Comment (hbelmiro) [inline quarkus/addons/persistence/rocksdb/deployment/src/main/java/org/kie/kogito/persistence/quarkus/rocksdb/KogitoAddOnPersistenceRocksDBProcessor.java:2]:
```suggestion
 * Copyright 2023 Red Hat, Inc. and/or its affiliates.
```

**Review by hbelmiro — APPROVED** [review]
LGTM.
I only left a comment regarding the year in a copyright comment.

**Pull request merged**